schemas/revision.yml

```diff
@@ -0,0 +1,48 @@
+required:
+  - patch
+
+properties:
+  id:
+    type: string
+    readOnly: true
+    description: |
+      The revisions's identifier (generated by the API)
+
+  url:
+    type: string
+    readOnly: true
+    description: |
+      API url for accessing the revision description.
+
+  user_id:
+    type: string
+    readOnly: true
+    description: |
+      The identifier for the user who created this revision.
+
+  created:
+    type: number
+    readOnly: true
+    description: |
+      Timestamp representing when the revision was created in milliseconds
```

Comment:
Good point, sounds like we need a revision number field to order by instead of creation time.